### PR TITLE
Basic layout for CoW Learn

### PR DESCRIPTION
**Pull request by anxolin**
Add basic layout for the blog. 

This is not meant to be a final design, as this will be fone by @fairlighteth . I just want to make a starting point that is better than the current page:

Current page:
image

## Renamed the page to Learn
image

## Added some categories
For now mocked. But added a grid of categories as the first section
image

## Show me the code
Adds a section to advertise the Tutorials pages (for devs)

image

## Latest articles
List articles srted by descendant publish date

image

## Get in touch section
At the end a section to get in touch

image

## Traverse entities
This PR also adds how to traverse entities

In this case, we want to get the author of each article, so we need to instruct strapi to traverse it.

This is how we can show:

image

## Not included
- For now ignore the post itself, this PR focuses mostly on adding some basic styles in the landing. @elena-zh please, don't make too deep test as Michel will move everything around
- The categories are mocked. They need to be properly implemented and loaded from the CMS in future PRs
- Pagination for articles
- Show the author

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @anxolin , awesome job!

If not to dig too deep (as you asked), I can mention these issues: 
1. Articles headers do not look good on a reduced screen 
image
2. DAO should be in capitals 
dao
3. To implement and fix issues with responsive views I created #200 issue. 
image
4. WDYT about making the whole article tab clickable, not only the header? (not sure about it)
5. If the page is renamed to 'Learn', I'd expect to see the link to it somewhere in the footer :) 
image

Thanks!

File: services/cms/index.ts [hunk 1]
```diff
@@ -3,8 +3,41 @@ import { PaginationParam } from "types";
...
+    return component.__component === 'SharedQuoteComponent'
+  }
+
+  export function isSharedRichTextComponent(component: ArticleBlock): component is SharedRichTextComponent {
+    return component.__component === 'SharedRichTextComponent'
+  }
+
+  export function isSharedSliderComponent(component: ArticleBlock): component is SharedMediaComponent {
+    return component.__component === 'SharedSliderComponent'
+  }
```
Comment (alfetopito) [inline services/cms/index.ts:37]: Nitpick: indentation

**Review by alfetopito — APPROVED** [review]
Not commenting on everything as there are many, not just in the PR but repo as a whole.
Just raising that the formatting style is not consistent.

File: services/cms/index.ts (hunk 1 above)
Comment (anxolin) [reply services/cms/index.ts:37]: When we move this to our mono, this will be automatically improved

**Comment by anxolin** [comment]
For 1. I improved a bit, is not bullet proof, but this will likely change a lot once @fairlighteth gets here, so I don't want to over do it

Screenshot at Apr 19 16-19-31
Screenshot at Apr 19 16-19-03
Screenshot at Apr 19 16-18-51

**Comment by anxolin** [comment]
@elena-zh 

1. Done to some degree (see comment above)
2. Done
3. Better to keep track on that issue. @fairlighteth will help with this
4. Not sure about this. I'll leave it up to @fairlighteth
5. Done

**Review by elena-zh — APPROVED** [review]
@anxolin , thank you!

Forgot to mention about some more improvements: 
1. Add 'Subscribe' button to the Stay tunned message (very complex)
2. Add search for article option (by a key word or so). 
Screenshot_3
 LMK please if any of these proposals is worth implementing, so I will open a task for this.

**Comment by anxolin** [comment]
@elena-zh Those are good ideas. You can maybe add an issue with a wish list. And after we are done, we can add some of those

**Comment by elena-zh** [comment]
> @elena-zh Those are good ideas. You can maybe add an issue with a wish list. And after we are done, we can add some of those

Opened #204 and #205 tasks for these proposals.

**Pull request merged**